lib/chocolatechip.js

```diff
@@ -202,18 +199,20 @@ global.ChocolateChip = {
   // MARK: Public
 
   eat(options) {
-    this._options = Object.assign({}, this._options, options);
+    const language = defaultTranslations[options.language] != null ? options.language : 'en_GB';
+    const translations = Object.assign({}, defaultTranslations[options.language], options.translations[options.language]);
```

Comment:
Please modify line 203 to be using the language variable directly instead of getting the variable from options.